### PR TITLE
Add more emphasis on local CMS and include some common gotchas in the README

### PR DESCRIPTION
**Pull request by paulanthonywilson**
Setting up for development documentation:

* Signpost setting up a local CMS a little more
* Adds a section for common gotchas and mitigations encountered while we Cultivars were getting set up on the 12 June 2017.

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1748)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +67,43 @@ $ foreman s
 
 ### Change CMS URL Path
 
-In development, frontend will use the local CMS for convenience. You can change the MAS_CMS_URL on .env file. Use http://cms.moneyadviceservice.org.uk to point to LIVE content, https://cms.qa.dev.mas.local for testing Or http://localhost:PORT to point to a local running CMS.
+In development, frontend will use the local CMS for convenience. See [CMS repository README](https://github.com/moneyadviceservice/cms/blob/master/README.md) for instructions on setting up a local CMS instance.
+
+You can change the MAS_CMS_URL on .env file. Use http://cms.moneyadviceservice.org.uk to point to LIVE content, https://cms.qa.dev.mas.local for testing Or http://localhost:PORT to point to a local running CMS.
```
Comment (norm) [inline README.md]: a) *https* for cms.moneyadviceservice.org.uk; and b) whilst I am not speaking authoritatively I would strongly argue that the live service should never be used against a development environment, so I would recommend removing it entirely :)

Comment (paulanthonywilson) [inline README.md]: @norm 👍  for https (I assume)

I can envisage pointing to the production CMS if there was a particularly puzzling issue that was only occurring in production. It's read-only, after all. 

That said, I have no strong opinions either way on documenting that here. Maybe worth checking with @pmcostello,  who added that suggestion, before deleting it?

Comment (paulanthonywilson) [inline README.md]: My mistake - Paul didn't add it, just changed the urls from the previous ones.

**Comment by shaley91** [comment]
Can we remove the ruby versions in Readme as they are not the same as the `.ruby_version` file.

File: README.md (hunk 1 above)
Comment (paulanthonywilson) [inline README.md]: @norm  Removed the reference to using production CMS in development.

Comment (norm) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -8,8 +8,7 @@ The Money Advice Service's responsive website.
 ## Prerequisites
 
 * [Git]
-* [Ruby 2.1.2][Ruby]
-* [Rubygems 2.2.2][Rubygems]
+* [Ruby][Ruby] - see verstion specified in [.ruby-version](.ruby-version)
```
Comment (Tr4pSt3R) [inline README.md]: ```see *verstion specified```

* _small_ spelling mistake

File: README.md [hunk 3]
```diff
@@ -67,10 +66,43 @@ $ foreman s
...
+```
+
+The CMS is setup but the database is empty. A solution is to copy the QA CMS database into the CMS, as detailed in the [CMS repository README](https://github.com/moneyadviceservice/cms/blob/master/README.md).
+
+#### Problems loading Dough or Yeast
+
+Assuming you have run `bowndler install`, you may have issues with previous `bower` installations.
+
+```
+rm -rf vendor/assets/bower_components
```
Comment (Tr4pSt3R) [inline README.md]: You can also remove the generated bower json file .i.e ```rm -f bower.json```.

Comment (paulanthonywilson) [inline README.md]: Good call.

Why `-f` though?

File: README.md (hunk 2 above)
Comment (paulanthonywilson) [inline README.md]: duh!

File: README.md (hunk 3 above)
Comment (Tr4pSt3R) [inline README.md]: @paulanthonywilson you can do it with/without. I just want it comprehensively vanquished :)

**Comment by johnplayer1982** [comment]
Looks good to me :+1: nice work!

**Comment by paulanthonywilson** [comment]
thanks @Tr4pSt3R @johnplayer1982

**Pull request merged**